docs/frameworks/typesafe-project.md

```diff
@@ -13,10 +13,9 @@ import TabItem from '@theme/TabItem';
   <title>Getting Started with Genezio | Genezio Documentation</title>
 </head>
 
+Welcome to the Genezio Typesafe "Getting Started" guide! In this tutorial, you will learn how to write a backend class, export its methods, call them from the frontend code, and then deploy both the frontend and the backend locally for testing and pushing live to a staging environment.
 
-Welcome to the "Getting Started" guide! In this tutorial, you will learn how to write a backend class, export its methods, call them from the frontend code, and then deploy both the frontend and the backend locally for testing and pushing live to a staging environment.
-
-You can easily start with from a template project in the [Genezio Dashboard](https://app.genez.io/new-project/).
+You can easily start with from a template project in the [Genezio Dashboard](https://app.genez.io/auth/login?redirect=new-project).
```

Comment:
```suggestion
:::tip
Get started in no time with the [Typesafe Project template](https://app.genez.io/auth/login?redirect=new-project).
:::
```

Reply:
de ce punem link catre login page cu redirect? Normal ar fi sa punem link catre pagina din spate si va face ea redirect daca nu esti logat, nu?

Reply:
we don't have this in place